guides/common/modules/proc_refreshing-clients-of-insights-iop.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="refreshing-clients-of-insights-iop"]
+= Refreshing clients of {insights-iop}
+
+[role="_abstract"]
+If you use {insights-iop}, you must refresh {insights-iop} registration of your hosts to keep using services of {insights-iop} after upgrading {ProjectServer}.
+
+.Prerequisites
+* You have root access on your hosts.
+
+.Procedure
+* Refresh `insights-client` registration on your hosts, for example by using remote execution:
```

Comment:
```suggestion
* On your hosts, refresh the `insights-client` registration:
+
```

And then after line 17

````
+
You can use remote execution to run this command on your hosts.
For more information, see X.
````

What do you think?

Reply:
I like the sound of this, LGTM otherwise